content/en/blog/2019/introducing-istioctl-analyze/index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Introducing istioctl analyze
+description: Analyze your Istio configuration to detect potential issues and get general insights.
+publishdate: 2019-11-14
+subtitle:
+attribution: David Ebbo (Google)
+keywords: [debugging,istioctl,configuration]
+target_release: 1.4
+---
+
+{{< boilerplate experimental-feature-warning >}}
```

Comment:
I don't think this banner is appropriate in a blog. It should be in the actual doc, which it is. In the blog just say `... introduces an experimental new tool to help you analyze ...`

(Side note: we generally avoid using words like `powerful`)